PWGHF/D2H/Tasks/taskLcToK0sP.cxx

```diff
@@ -39,14 +39,14 @@ struct HfTaskLcToK0sP {
   HistogramRegistry registry{
     "registry",
     {// data
-     {"hMass", "cascade candidates;inv. mass (#pi K) (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{500, 0.0f, 5.0f}}}},
+     {"hMass", "cascade candidates;inv. mass (p K_{S}^{0}) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{500, 1.5f, 3.0f}, {30, 0.0f, 30.0f, "#it{p}_{T} (GeV/#it{c})"}}}},
```

Comment:
- Your invariant mass binning should still be finer (or just use the range 1.98--2.58 like Lc->pKpi)
- `entries` should be `p_{T}`
- Suggestion to have a configurable pT binning, see https://github.com/AliceO2Group/O2Physics/blob/41eed29/PWGHF/D2H/Tasks/taskLc.cxx#L39 + https://github.com/AliceO2Group/O2Physics/blob/41eed29/PWGHF/D2H/Tasks/taskLc.cxx#L137

Reply:
OK, I changed that. I tried using `ConfigurableAxis`, but I did not manage to set the default with `hf_cuts_lc_to_k0s_p::vecBinsPt`, so I went for the same option as in https://github.com/AliceO2Group/O2Physics/blob/41eed29/PWGHF/D2H/Tasks/taskLc.cxx#L39 with `Configurable<std::vector<double>>`. I did however not need the extra step as in  https://github.com/AliceO2Group/O2Physics/blob/41eed29/PWGHF/D2H/Tasks/taskLc.cxx#L137    , but could just use `binsPt` directly for the declaration of the axes.